src/iso9660.rs

```diff
@@ -298,13 +313,22 @@ impl PrimaryVolumeDescriptor {
             parse_iso9660_string(eat(buf, 1), 32, IsoString::StrA).context("parsing system id")?;
         let volume_id = // technically should be StrD, but non-compliance is common
             parse_iso9660_string(buf, 32, IsoString::StrA).context("parsing volume id")?;
-        let root = match get_next_directory_record(eat(buf, 156 - 72), 34, true)? {
+        eat(buf, 8); // Unused field always 0x00
+        let volume_space_size = // Data type: int32_LSB-MSB
```

Comment:
The comment isn't really necessary, since it's clear from context.  I'm okay with keeping it, but let's put it at the end of the statement, not after the `=`.